backend/app.js

```diff
@@ -1,7 +1,36 @@
+//Getting express and mongoose before setting up the server with 'app ='
 const express = require('express')
+const mongoose = require('mongoose')
+const bodyParser = require('body-parser')
 const app = express()
 var cors = require('cors')
 
+//add promise library
+mongoose.Promise = require('bluebird');
```

Comment:
In a follow up PR we should move all mongo setup in general to a separate file, and import as necessary.